### DISCUSSION TITLE
Bullhorn-HubSpot Integration

### DISCUSSION THREAD

**Discussion by CIBR1** [bullhorn/career-portal#454] [Q&A]
Hi, we're using the Bullhorn app in HubSpot to import contacts from Bullhorn to HubSpot. The problem is it doesn't assign the correct contact owner and sometimes it assigns the contact to a random hubspot user. Each contact in bullhorn has their owner assigned and each of those owners have their own corresponding hubspot user account.
![image](https://user-images.githubusercontent.com/95701874/146584098-10253b83-2dfd-4c3a-a270-e66685fc8fdb.png)

Would appreciate any help. Thank you!

**Comment by charlesabarnes** [comment]
Hello, this is not related to the career portal project front end code.  I suggest starting off by reaching out to Bullhorn Support or whoever the creator of the Bullhorn to Hubspot sync software is
